File: HorseRacing.Contracts/Models/Game/Dtos/HorseBetDto.cs
using HorseRacing.Contracts.Base.Dto;

namespace HorseRacing.Contracts.Models.Game.Dtos
{
    public class HorseBetDto : BaseDto
    {
        /// <summary>
        /// Масть
        /// </summary>
        public int BetSuit { get; set; }
        /// <summary>
        /// Сумма ставки
        /// </summary>
        public int BetAmount { get; set; }
        /// <summary>
        /// Код пользователя
        /// </summary>
        public Guid UserId { get; set; }
        /// <summary>
        /// ФИО (логин)
        /// </summary>
        public string FullName { get; set; } = string.Empty;
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/dto/horse-bet-dto.ts
import type { BaseDto } from '~/interfaces/api/contracts/base/dto/base-dto';
import type { SuitType } from '../enums/suit-type-enum';

export interface HorseBetDto extends BaseDto {
    /** Масть */
    BetSuit: SuitType;
    /** Сумма ставки */
    BetAmount: number;
    /** Код пользователя */
    UserId: string;
    /** ФИО (логин) */
    FullName: string;
}
